third_party/2and3/redis/client.pyi

```diff
@@ -277,6 +277,10 @@ class Redis(object):
     def script_kill(self): ...
     def script_load(self, script): ...
     def register_script(self, script): ...
+    def pubsub_channels(self) -> List[Text]: ...
+    def pubsub_numsub(self, *args) -> List[Tuple[Text, int]]: ...
```

Comment:
`args` are the channel names, so it could get a `Text` annotation.

Reply:
Changes applied.